app/utils/prepareAnswer.js

```diff
@@ -18,6 +18,17 @@ function prepareStatsCompactAnswer(body, index, nameState) {
         data += '<i> (+' + numberWithIndianCommas(statedata['deltaconfirmed']) + ')</i>'
     }
     data += "\n\u{1F7E0} Active: " + numberWithIndianCommas(statedata['active'])
+    
+    let deltaActive = parseInt(statedata['deltaconfirmed']) - parseInt(statedata['deltarecovered']) - parseInt(statedata['deltadeaths'])
+//change in active number will not be displayed if it is zero.
+ if (deltaActive > 0)
+     data += '<i> (+' + numberWithIndianCommas(deltaActive) + ')</i>'
+ else if (deltaActive < 0) {
+     //required to display change in active numbers correctly if active number decreases
+     deltaActive = deltaActive + (-2) * deltaActive
```

Comment:
I think this line is unnecessary.
You could just do `Math.abs(deltaActive)` to get the positive number and further add - yourself as a string.

Reply:
good suggestion 👍 ,i will change it

Reply:
done 💯